src/pybind/pynmodl.cpp

```diff
@@ -171,10 +182,14 @@ class JitDriver {
     benchmark::BenchmarkResults run(std::shared_ptr<nmodl::ast::Program> node,
                                     std::string& modname,
                                     int num_experiments,
-                                    int instance_size) {
+                                    int instance_size,
```

Comment:
shoudln't this be conditional on running with GPU enabled?
I would add the grid size into the CodegenConfig maybe? or maybe into an additional constructor for JitDriver that is in a `#ifdef NMODL_LLVM_CUDA_BACKEND`.

Reply:
I think that the grid dimensions conceptionally shouldn't be part of `CodegenConfig` since they don't influence in any way the code generation.
What we could do is create another declaration of `run` function with these parameters only `#ifdef NMODL_LLVM_CUDA_BACKEND` however this would duplicate some parts of the code so maybe it's better to just keep those parameters as optional and just in the CPU case they are not taken into consideration

Reply:
OK, that's fine with me. It'll need maybe still some cleanup at a later point.